### ISSUE TITLE
Numerous occurrences of "RuntimeError: Could not connect with the Tribler Core"

### ISSUE THREAD

**Issue by devos50** [Tribler/tribler#5403]
I noticed numerous occurrences of the "RuntimeError: Could not connect with the Tribler Core within 60 seconds" error. These errors have been exclusively been reported by Linux users (e.g., Debian, Ubuntu system).

We should install and run Tribler in a clean Linux environment to see if we are missing any dependency.

**Comment by devos50** [comment]
Just checked with a fresh Ubuntu 20.04 installation; it seems to work out of the box.

**Comment by ichorid** [comment]
It only happened with python 3.6.9 according to reports. We should update the required Python version (ipv8 already requires 3.7, so that should be the problem).

**Comment by devos50** [comment]
It seems that we are using `pyinstaller` to create the `.deb` file (not sure when this was changed actually). The resulting `.deb` therefore contains the full Python interpreter and thus does not use the system Python version. I think this explains why all the reports show Python 3.6.9.

That said, we should consider building with a higher Python version (e.g., 3.8).

**Comment by xoriole** [comment]
Yes, I actually made the switch to PyInstaller based Debian builder because we had issues with python packages that were not available in APT. We did have a workaround by installing pip first and then packages with pip but that had issues messing up user python/pip installation. Using PyInstaller for linux provided the same flexibility as macOS and Windows in terms of having a new python dependency.

However, the issue with traceback needs to be fixed. Maybe first the python version on the builder could be updated to 3.8.
